app/frontend/src/features/auth/useAuthStyles.ts

```diff
@@ -20,11 +21,25 @@ const useAuthStyles = makeStyles((theme) => ({
   },
   button: {
     marginTop: theme.spacing(4),
+    [theme.breakpoints.up("md")]: {
+      borderRadius: "4px",
+      width: "180px",
```

Comment:
Remove this and use `alignItems: "flex-start"` in the parent (you may have to fix some other flex containers)